app/controllers/events_controller.rb

```diff
@@ -0,0 +1,45 @@
+class EventsController < ApplicationController
+  before_action :set_post, only: [:show, :edit, :update, :destroy]
+  before_action :logged_in_user, only: [:create, :new, :show]
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = current_user.events.build(event_params)
+    if @event.save
+      flash[:success] = "Event Created Successfully"
+      redirect_to root_url
+    else
+      render 'new'
+    end
+  end
+
+  def index
+    @events = Event.all
+    @past = Event.past
+    @future = Event.upcoming
+  end
+
+  def show
+    #@event = Event.find(params[:id])
+  end
+
+  def attend
```

Comment:
Placing `attend` action here isn't right for two reasons:

1. `attend` isn't a CRUD action as far as the `events_controller` is concerned. By convention (which we need to abide by if we wanna be good developers), we only put CRUD actions in CRUD resources. 

2. In relation to 1) `attend` seems to be creating an attendance record. So why didn't we just make this as a `create` action in `attended_events_controller` (which you don't have)?